lib/pa11y-ci.js

```diff
@@ -115,7 +116,7 @@ function pa11yCi(urls, options) {
 		function testRunComplete() {
 			const passRatio = `${report.passes}/${report.total} URLs passed`;
 
-			testBrowser.close();
+			testBrowsers.forEach(testBrowser => testBrowser.close());
```

Comment:
Could this unintentionally close a browser for which another test is still running?

Reply:
Good question, looked at `async/queue` and the `drain` method is only triggered once all tasks in the queue are complete so this should only run at the end.